### ISSUE TITLE
Time Resolved e L-1 s-1

### ISSUE THREAD

**Issue by GitHubDoug** [FundyPhytoPhys/PicoDiel#1]
- Decide which MC files to use; filtering already done upstream; Mireille & Sylwia
- MC data has PAR trace, which can be validated with calibration if we want
- MC data has deltaOD, OD680 and OD720 every 5 min

- Sylwia is generating OD720 compared to pico cell countsfor exponential phase cultures; convert OD720 to cell counts
- (Could be done for Prochlorococcus?)

- Sylwia is generating Turner Chlorophyll compared to delta OD and OD680 for exponential phase cultures; convert deltaOD or OD680 to [Chl]
   Most measurements taken in afternoon, in parallel Solisense JVPSII, Turner Chlorophyll (triplicate, summarize) and pico cell counts.

- Doug calibrated e L-1 s-1 from JVPSII and O2 data from Sylwia Baltics light response curves
- Could be done for Prochloroccocus
- Doug fit e L-1 s-1 vs. light with Platt model, get alpha, Pmax and beta
- Sylwia is generating e chl-1 s-1 and e cell-1 s-1 for exponential phase cultures by normalizing e L-1 s-1 to chl L-1 or cells L-1
- Could be done for Prochlorococcus
- Fit e- chl-1 s-1 vs. light, and/or e- cell-1 s-1 vs. light, using above calibrations
- Import LRC curve fit parameters for e- chl-1 s-1 and e- cell-1 s-1
- Generate time resolved e- chl-1 s-1 and e- cell-1 s-1 using short term light,  deltaOD and/or OD720 from MC data traces
- Compare short term growth rates to short term e- chl-1 s-1 and e- cell-1 s-1
